packages/components/src/internal/components/navigation/ProductMenu.tsx

```diff
@@ -279,11 +282,14 @@ interface ProductMenuButtonTitle {
     routes: any[];
 }
 
+const HOME_PATH = '/home';
+const HOME_TITLE = 'Home Project';
+
 export const ProductMenuButtonTitle: FC<ProductMenuButtonTitle> = memo(props => {
     const { container, folderItems, routes } = props;
     const title = useMemo(() => {
-        return folderItems?.length > 1 ? container.title : 'Menu';
-    }, [container.title, folderItems?.length]);
+        return folderItems?.length > 1 ? (container.path === HOME_PATH ? HOME_TITLE : container.title) : 'Menu';
```

Comment:
Is this because the title comes back as "home" (lower case)?

Reply:
Yes, without this it was just saying "home". Request from Andrew was to use "Home Project".